README.md

```diff
@@ -72,11 +61,9 @@ $ oc create role use-scc-hostmount-anyuid --verb=use --resource=scc --resource-n
 $ oc adm policy add-role-to-user use-scc-hostmount-anyuid system:serviceaccount:$NAMESPACE:nfs-client-provisioner
 ```
 
-**Step 4: Configure the NFS-Client provisioner**
-
-Note: To deploy to an ARM-based environment, use: `deploy/deployment-arm.yaml` instead, otherwise use `deploy/deployment.yaml`.
+**Step 4: Configure the NFS subdir external provisioner**
 
-You must edit the provisioner's deployment file to specify the correct location of your nfs-client-provisioner container image.
+You must edit the provisioner's deployment file to specify the correct location of your nfs-subdir-external-provisioner container image.
```

Comment:
i kept this as `nfs-client-provisioner` because it mention the container image name which is still nfs-client-provisioner in the yamls

```yaml
      containers:
        - name: nfs-client-provisioner
```